Makefile.core.mk

```diff
@@ -356,9 +333,13 @@ BINARIES:=./istioctl/cmd/istioctl \
 RELEASE_BINARIES:=pilot-discovery pilot-agent sidecar-injector mixc mixs mixgen node_agent node_agent_k8s istio_ca istioctl galley sdsclient
 
 .PHONY: build
-build: depend
-	STATIC=0 GOOS=$(GOOS) GOARCH=$(GOARCH) LDFLAGS='-extldflags -static -s -w' common/scripts/gobuild.sh $(ISTIO_OUT)/ $(BINARIES)
+build: depend build-linux
```

Comment:
I want to test this more locally, but won't this build twice on linux?Of course the second time is an incremental build so its pretty quick

Reply:
yes that is the intent.  We build twice - once for the target arch - and once for the container. If they are the same arch, there is no need to build them twice. An optimization I plan to make once I start testing Mac.

Reply:
confirmed this is building 2x on linux. Would be ideal to not have this twice but its not too bad

Reply:
I think this is a must fix before merge